feature/home/presentation/viewmodel/src/main/java/com/chocolatecake/viewmodel/people/PeopleDetailsUiState.kt

```diff
@@ -0,0 +1,35 @@
+package com.chocolatecake.viewmodel.people
+
+import com.chocolatecake.viewmodel.common.model.MediaVerticalUIState
+
+
+data class PeopleDetailsUiState(
+    val peopleData :PeopleDataUiState =PeopleDataUiState(0,"","","","","","","") ,
+    val Movies: List<MediaVerticalUIState> = emptyList(),
+    val TvShows: List<MediaVerticalUIState> = emptyList(),
+    val onErrors: List<String> = emptyList(),
+    val isLoading: Boolean = false
+) {
+
+//    data class MoviesUiState(
+//        val id: Int,
+//        val imageUrl: String,
+//        val rate: Double
+//    )
+//
+//    data class TvShowsUiState(
+//        val id: Int,
+//        val imageUrl: String,
+//        val rate: Double
+//    )
+
+    data class PeopleDataUiState(
```

Comment:
change name to PersonInfoUiState

Reply:
PeopleDataUiState is only used in the data class PeopleDetailsUiState so add it in its body instead